app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,13 @@
+Use My Current Location:<br>
+
+Travel Distance:<br>
+<%= text_field(:post, :radius) %> Miles<br><br><br>
+
+
+Price Range:<br>
+
+<% form_for @entry do |f| %>
```

Comment:
Potentially going to be removed as this functionality has been replaced already.